DigitalLearningSolutions.Web/Views/TrackingSystem/CourseSetup/AdminFields/AdminFields.cshtml

```diff
@@ -0,0 +1,58 @@
+﻿@inject IConfiguration Configuration
+@using DigitalLearningSolutions.Web.ViewModels.TrackingSystem.CourseSetup
+@using Microsoft.Extensions.Configuration
+@model AdminFieldsViewModel
+
+<link rel="stylesheet" href="@Url.Content("~/css/shared/cardWithButtons.css")" asp-append-version="true">
+
+@{
+  ViewData["Title"] = "Manage Course Admin Fields";
+  ViewData["Application"] = "Tracking System";
+  ViewData["HeaderPath"] = $"{Configuration["AppRootPath"]}/TrackingSystem/Centre/Dashboard";
+  ViewData["HeaderPathName"] = "Tracking System";
+  var canAddNewField = Model.AdminFields.Count() < 3;
+}
+
+@section NavMenuItems {
+    <partial name="/Views/TrackingSystem/Shared/_NavMenuItems.cshtml" />
+  }
+
+@section NavBreadcrumbs {
+    <partial name ="/Views/TrackingSystem/Shared/_CentreCourseSetupBreadcrumbs.cshtml"/>
+  }
+
+<div class="nhsuk-grid-row">
+  <div class="nhsuk-grid-column-full">
+
+    <h1 class="nhsuk-heading-xl">Manage course admin fields</h1>
+
+    <p class="nhsuk-lede-text nhsuk-u-margin-top-4">
+      Optional additional fields to store useful data against delegate course progress.
+      This data can be managed by administrators in the Course Delegates view.
+      To request additions to the list of available fields, please raise a ticket.
+    </p>
+
+    @foreach (var field in Model.AdminFields) {
+      <partial name="_AdminFieldCard.cshtml" model="field"/>
+    }
+
+  </div>
+</div>
+
+@if (canAddNewField) {
+  <div class="nhsuk-grid-row">
+    <div class="nhsuk-grid-column-full">
+      <a class="nhsuk-button" role="button" href="#">Add admin field</a>
+    </div>
+  </div>
+}
+
+<div class="nhsuk-grid-row nhsuk-u-margin-top-7">
+  <div class="nhsuk-grid-column-full">
+    <vc:back-link asp-controller="" asp-action="" link-text="Go back"/>
+  </div>
+</div>

```

Comment:
Can we structure the html here in a similar way to the delegate prompts index page for consistency or is there a good reason not to?